app/src/main/java/com/gruppo4/SMSApp/MainActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.gruppo4.SMSApp;
```

Comment:
There is some Italian here, consider using `com.group4.SMSApp` as the company name or some code representing your group.

Reply:
yeah... But I don't think that's gonna be possible right now.
We created SMSApp with that package name in the very beginning... In next updates of the Master the name would eventually change, and my branch would eventually die... After an update, I'll be able to drag and drop the package audioUtility into the new one, "killing" the old gruppo4.SMSApp package.
I can't do it now, since it's shared, and I don't want to create some troubles if I want to ask my colleagues to merge.
But thank you for pointing this out. I'll talk with my colleagues about this.

Reply:
Ok, I understand your situation. This is not a big issue so feel free to fix it when it is safe to do.